bin/latex.py

```diff
@@ -164,7 +165,7 @@ def build_latex_pdf(builddir, tex_path, problem_path=None):
     )
 
     if ret.ok is not True:
-        print(f'{Fore.RED}Failure compiling pdf:{Style.RESET_ALL}\n{ret.out}')
+        print(f'{Fore.RED}Failure compiling pdf:{Style.RESET_ALL}\n{ret.out}', file=sys.stderr)
```

Comment:
Maybe some of the `print(..., file=sys.stderr)` can be replaced with calls to the logger functions of `util.py`? For example, in this case, a call to `error` could be used, and line 178 in this file could be a call to `log`.

Reply:
done - the remaining uses for `Fore` and `Style` all do some kind of custom styling so can't be replaced by log/error.